scripts/queuestat

```diff
@@ -294,38 +294,27 @@ class Queuestat(object):
                                    ns_diff(cntr['totalbytes'], old_cntr['totalbytes']) != '0' or \
                                    ns_diff(cntr['droppacket'], old_cntr['droppacket']) != '0' or \
                                    ns_diff(cntr['dropbytes'], old_cntr['dropbytes']) != '0' or \
                                    ns_diff(cntr['creditWDpkts'], old_cntr['creditWDpkts']) != '0':
-                       table.append((port, cntr['queuetype'] + str(cntr['queueindex']),
+                        table.append((port, cntr['queuetype'] + str(cntr['queueindex']),
                                    ns_diff(cntr['totalpacket'], old_cntr['totalpacket']),
                                    ns_diff(cntr['totalbytes'], old_cntr['totalbytes']),
                                    ns_diff(cntr['droppacket'], old_cntr['droppacket']),
                                    ns_diff(cntr['dropbytes'], old_cntr['dropbytes']),
                                    ns_diff(cntr['creditWDpkts'], old_cntr['creditWDpkts'])))
-                   elif not non_zero or cntr['totalpacket'] != '0' or cntr['totalbytes'] != '0' or \
-                                  cntr['droppacket'] != '0' or cntr['dropbytes'] != '0' or cntr['creditWDpkts'] != '0':
-                       table.append((port, cntr['queuetype'] + str(cntr['queueindex']),
-                                  cntr['totalpacket'], cntr['totalbytes'],
-                                  cntr['droppacket'], cntr['dropbytes'], cntr['creditWDpkts']))
                 else:
-                   if not non_zero or ns_diff(cntr['totalpacket'], old_cntr['totalpacket']) != '0' or \
+                    if not non_zero or ns_diff(cntr['totalpacket'], old_cntr['totalpacket']) != '0' or \
                                    ns_diff(cntr['totalbytes'], old_cntr['totalbytes']) != '0' or \
                                    ns_diff(cntr['droppacket'], old_cntr['droppacket']) != '0' or \
                                    ns_diff(cntr['dropbytes'], old_cntr['dropbytes']) != '0':
-                      table.append((port, cntr['queuetype'] + str(cntr['queueindex']),
-                                  ns_diff(cntr['totalpacket'], old_cntr['totalpacket']),
-                                  ns_diff(cntr['totalbytes'], old_cntr['totalbytes']),
-                                  ns_diff(cntr['droppacket'], old_cntr['droppacket']),
-                                  ns_diff(cntr['dropbytes'], old_cntr['dropbytes'])))
-                   elif not non_zero or cntr['totalpacket'] != '0' or cntr['totalbytes'] != '0' or \
-                                     cntr['droppacket'] != '0' or cntr['dropbytes'] != '0':
-                      table.append((port, cntr['queuetype'] + str(cntr['queueindex']),
-                               cntr['totalpacket'], cntr['totalbytes'],
-                               cntr['droppacket'], cntr['dropbytes']))
-
+                       table.append((port, cntr['queuetype'] + str(cntr['queueindex']),
```

Comment:
Could you add a unit test to verify the logic change here?

Reply:
> Could you please add a unit test to verify the logic change here?

There are tests already in https://github.com/sonic-net/sonic-utilities/blob/master/tests/queue_counter_test.py which covers these changes.